### ISSUE TITLE
Fix failing FocusTracker tests in Firefox.

### ISSUE THREAD

**Issue by afshin** [jupyterlab/lumino#21]
The `FocusTracker` tests (in `focustracker.spec.ts`) pass in CI and in Chrome, but not in Firefox.

**Comment by pplonski** [comment]
Hi! I was trying to reproduce this with following steps:

```
git clone https://github.com/jupyterlab/lumino.git
cd lumino
yarn
cd packages/widgets
yarn build
cd -
yarn clean
yarn build
yarn build:test
yarn test
```

But it looks like test cant find the `bundle.test.js` file for all packages. The beginning of the test output:

```
yarn run v1.21.1
$ lerna run test
lerna notice cli v3.19.0
lerna info versioning independent
lerna info Executing command in 16 packages: "yarn run test"
lerna info run Ran npm script 'test' in '@lumino/algorithm' in 3.9s:
$ npm run test:firefox

> @lumino/algorithm@1.2.2 test:firefox /home/piotr/sandbox/jl/lumino/packages/algorithm
> cd tests && karma start --browsers=Firefox


START:
19 12 2019 22:01:45.112:WARN [filelist]: Pattern "/home/piotr/sandbox/jl/lumino/packages/algorithm/tests/build/bundle.test.js" does not match any file.
19 12 2019 22:01:45.127:INFO [karma-server]: Karma v4.4.1 server started at http://0.0.0.0:9876/
19 12 2019 22:01:45.127:INFO [launcher]: Launching browsers Firefox with concurrency unlimited
19 12 2019 22:01:45.131:INFO [launcher]: Starting browser Firefox
19 12 2019 22:01:47.825:INFO [Firefox 71.0.0 (Ubuntu 0.0.0)]: Connected on socket 1oRhntqybNuKnk1WAAAA with id 60825187

Finished in 0.001 secs / 0 secs @ 22:01:48 GMT+0100 (Central European Standard Time)

SUMMARY:
✔ 0 tests completed
```

**Comment by vidartf** [comment]
@pplonski Your steps seem to differ from those in the contributing guide: https://github.com/jupyterlab/lumino/blob/master/CONTRIBUTING.md Does it work if you follow that?

**Comment by pplonski** [comment]
@vidartf thank you for help. I've followed the steps from the link you pointed but it also didn't work. I think I've found the reason. For example in dir `packages/widgets` the file `bundle.test.js` is not in `tests/build` directory but in `tests/dist/build` when I change the path in `karma.conf.js` then I can run tests. But still can't reproduce the problem with failing FocusTracker tests in Firefox. After running:

```
# running in packages/widgets directory
yarn test:firefox
```

I got:

```
SUMMARY:
✔ 794 tests completed
Done in 6.19s.
```

I have 2 questions:
1. Is the configuration ok in all packages, and there is a problem on my side that `bundle.test.js` is in the wrong directory?
2. I can't reproduce the failing test on firefox. Is it a problem on my side that all tests work?
 (my Firefox is v71.0)

**Comment by telamonian** [comment]
I saw some very similar failures a couple of months ago: https://github.com/phosphorjs/phosphor/issues/441#issue-507427386

I can confirm that on my machine (OS X 10.14, Firefox 71) I still see the same set of 36 test failures in `focustracker.spec.ts` and `tabbar.spec.ts` in the latest Lumino master.

**Comment by telamonian** [comment]
@pplonski I was able to reproduce the pathing issue you're seeing with the tests. It turns out to be due to the upgrade from Webpack `v2` => `v4` that was made in #31. So currently while all of our tests build, none of them will run, even though the CI completely successfully.

There's a fix up at #35

**Comment by fcollonval** [comment]
Closing as obsolete - please open a new issue in case it comes back
